### DISCUSSION TITLE
no peer harvester peer info anymore in the log of chia116 ?

### DISCUSSION THREAD

**Discussion by mting806** [Chia-Network/chia-blockchain#5772] [Support]
hello
I am using multi harvester setup and using debug log to monitor the peer-farmer connection status.
after update to chia116 , the peer-farmer info is missing?
how could I check their connection?
thank you very much

[time stamp] farmer farmer_server   : INFO   -> new_signage_point_harvester to peer [harvester IP address] [peer id - 64 char hexadecimal]
[time stamp] farmer farmer_server   : INFO   <- farming_info from peer [peer id - 64 char hexadecimal] [harvester IP address]
[time stamp] farmer farmer_server   : INFO   <- new_proof_of_space from peer [peer id - 64 char hexadecimal] [harvester IP address]

**Comment by mting806** [comment]
found it, these content have been moved to debug level

**Reply by metoos003** [reply]
In the debug.log?

**Reply by metoos003** [reply]
I also encountered this problem. In 1.1.5, I will be prompted to find the harvester, but in 1.1.6, I can't see the harvester agreement. Does this have an impact?

**Comment by mting806** [comment]
yes 
before 116 , the log level = INFO will show them 
but in 116 you need to set log level = DEBUG to see them
and log files all are same debug.log

**Reply by metoos003** [reply]
Where do you change it?

**Comment by mting806** [comment]
~/.chia/mainnet/config/config.yaml 

log_level: DEBUG

**Reply by metoos003** [reply]
Thank you

**Comment by BrandtH22** [comment]
@mting806 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
